docker/ravendb-ubuntu-arm/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM arm32v7/ubuntu:bionic
```

Comment:
We use https://hub.docker.com/_/microsoft-dotnet-core-runtime-deps/ as a base now. There are arm32 and arm 64 bionic images available. I think we should use them instead.

Reply:
Also if we're adding support for arm32 we should probably also think of 64. So instead of `ravendb-ubuntu-arm` we should say `arm32v7` etc.

Reply:
Umm cross that. We only need `arm32v7`, we do *not* support `arm64`.